chained.go

```diff
@@ -0,0 +1,63 @@
+/*
+ *
+ * Copyright 2017 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package goat
```

Comment:
Don't know if we have a policy on this in go but the line lengths here are longer than 100 chars.

Reply:
hmm, I should fix my editor I suppose

I think in my defence, this change is a bit special in that it's just copy-pasting code from a high quality trusted source (grpc-go) into ours, so I've perhaps taken less time with testing and formatting than I would otherwise

I agree it's pretty weird the code is dangling without any use even in a unit test. It really is naughty of me... I'll look at that now

Reply:
Thanks. Sometimes I think reading unit tests help one understand the API